src/ui/popup/components/link.tsx

```diff
@@ -0,0 +1,22 @@
+import {m} from 'malevic';
+
+interface LinkProps {
+    cls: string;
+    url: string;
+}
+
+function openTab(url: string) {
```

Comment:
Alternative can be written as 
`const openTab = (url: string) => chrome.tabs.create({url});`
As a one-liner :)

Reply:
No, this would create tabs already when the component loads, it must return a function. 
But it can be certainly written shorter, put directly into onclick instead, making it a zero-liner ;)